static/app/gettingStartedDocs/python/aiohttp.tsx

```diff
@@ -41,19 +41,16 @@ sentry_sdk.init(
       params.isProfilingSelected &&
       params.profilingOptions?.defaultProfilingMode !== 'continuous'
         ? `
-    # Set profiles_sample_rate to 1.0 to profile 100%
+    # Set profile_session_sample_rate to 1.0 to profile 100%
     # of sampled transactions.
```

Comment:
If I understand [the docs](https://docs.sentry.io/platforms/python/profiling/#sampling) correctly, `profile_session_sample_rate` is relative to profile sessions, not to sampled transactions. So, we would need to update this comment accordingly. Should also define profile sessions a bit better.

```suggestion
    # Set profile_session_sample_rate to 1.0 to profile 100%
    # of profile sessions.
```

Reply:
This comment likely applies to the other files changed here